packages/vaex-ml/vaex/ml/transformations.py

```diff
@@ -242,8 +250,12 @@ def transform(self, df):
         for i, feature in enumerate(self.features):
             for j, value in enumerate(self.uniques_[i]):
                 column_name = self.prefix + feature + '_' + str(value)
```

Comment:
I don't like `None` as name, I think it should be, `_NaN` or `_missing` to be consistent with what we use in vaex.